### PR TITLE
Improvement(example): change README and App.svelte props to destructu…

### PR DESCRIPTION
**Pull request by Neulhan**
Hello, my name is Neulhan and I use svelte these days.
I find this project in searching for some swiper thing in svelte.
I read your project's README.md and I found a simple improvement point about example.

```
  let autoplay = false;
  let delay = 2000; //ms
  let showIndicators = true;
  let transitionDuration = 1000; //ms
  let defaultIndex = 0; //start from 0
.
.
.

<Swipe
    {showIndicators}
    {autoplay}
    {delay}
    {transitionDuration}
    {defaultIndex}
  >
```   
```
const swipeConfig = {
    autoplay: false,
    delay: 2000,
    showIndicators: true,
    transitionDuration: 1000,
    defaultIndex: 0,
  };
.
.
.
<Swipe {...swipeConfig}>
```  

I think ES6 destructuring is more simple. So I change examples.
Can you merge this change? I checked it works properly.

Also, I like your project very much. I want to give more contribute to your project something like Korean translation for example and README. Even if this pull request is rejected.
If you say it's okay, can I try and send you a pull request about Korean translation next time?

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,2 +1,2 @@
 export { default as Swipe } from "./Swipe.svelte";
-export { default as SwipeItem } from "./SwipeItem.svelte";
```
Comment (SharifClick) [inline src/index.js:2]: Leave that file untouched

**Comment by SharifClick** [comment]
Thanks for your PR @Neulhan , left few comments on that.
and contributions are always welcomed.
Cheers.

**Comment by Neulhan** [comment]
I exclude `index.js` in pull request as you said! And I will prepare Korean translation soon. See you later!

**Comment by Neulhan** [comment]
Uh oh, My visual studio formatter do that thing.
I kill all other things that formatted unnecessarily.

**Pull request merged**